index.js

```diff
@@ -76,6 +78,13 @@ async function main () {
     const debounced = debounceify(bound)
     list.core.on('append', debounced)
     await debounced()
+
+    swarm.firewall = async (remotePublicKey) => {
+      const allowedPeers = await list.getProperty(SeedBee.ALLOWED_PEERS_METADATA_KEY)
+      if (allowedPeers === '*' || allowedPeers === null) return false
+      if (allowedPeers.split(',').find(e => b4a.from(e.trim(), 'hex').equals(remotePublicKey))) return false
```

Comment:
The .equals can also be `b4a.equals()`, else we aren't fully compat.
Can also be simplified a bit by turning the conversions around:
```
const hexKey = b4a.toString(remotePublicKey, 'hex')
return !allowedPeers.includes(remotePublicKey)
```